plaidml2/core/ffi.cc

```diff
@@ -51,6 +58,20 @@ void plaidml_init(plaidml_error* err) {
           el::Loggers::setVerboseLevel(level);
         }
       }
+      auto ctx = GlobalContext::getContext();
+      auto eventlog_str = vertexai::env::Get("PLAIDML_EVENTLOG_FILENAME");
+      if (ctx && eventlog_str.size()) {
+        IVLOG(1, "Logging events to " << eventlog_str);
+        auto config = "{\"@type\": \"type.vertex.ai/vertexai.eventing.file.proto.EventLog\", \"filename\": \"" +
+                      eventlog_str + "\"}";
+        auto pconfig = vertexai::ParseConfig<google::protobuf::Any>(config);
+        auto eventlog = vertexai::AnyFactoryMap<vertexai::context::EventLog>::Instance()->MakeInstance(*ctx, pconfig);
+        ctx->set_eventlog(std::move(eventlog));
+        ctx->set_is_logging_events(true);
+      } else {
+        ctx->set_is_logging_events(false);
+        ctx->set_eventlog(std::shared_ptr<vertexai::context::EventLog>());
+      }
```

Comment:
Actually, can we just construct a protobuf object directly without having to parse some config string?